### DISCUSSION TITLE
Next Update?

### DISCUSSION THREAD

**Discussion by tarogers2000** [OS4ED/openSIS-Classic#321] [Ideas]
Hello,
When will we see an update to the system? I would love to see some of the features that the main opensis has in the community addition. I would love to see a Discipline Module, Graduation Progress, and when planning out a new school year having a visual scheduler with the teacher names on the side and a course search bar, subject, max students, etc on the top as well as then select which periods you want that course created and the periods and semesters selected it creates individual sections of that course with that teacher?. I have also seen the main OpenSIS has a new look to it that I think the community should get as well.

Thank you and have a wonderful day

**Comment by openSISAdmin** [comment]
Those features are part of the paid edition and will stay that way.

If you like the new openSIS SaaS product you should subscribe to it. We have priced it very low so that all schools can afford it.
